requirements.txt

```diff
@@ -1,3 +1,3 @@
 Flask==0.8
 Markdown==2.1.1
-plex==2.0.0dev
```

Comment:
sorry, this was a mistake. We inherited this from another fork which had initiated the work to make python3 compatible.